### PR TITLE
feat(console): add auth & project tutos + glossary

### PR DESCRIPTION
**Pull request by Nuttymoon**
### Changes

- `Console` section
  - Add `Glossary` page
  - Add `Tutorials`: `Authentication` and `Project Management`

### Additional comments

Tutorials are provided for the Ash CLI + Ash Console API

### CODE REVIEW

File: docs/console/tutorials/authentication.md [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+ash console auth refresh-token
+```
+
+```bash title="Output"
+Refreshing access token for the Ash Console at http://localhost:8080
+
+Access token refreshed successfully!
+```
+
+:::
```
Comment (leopaul36) [inline docs/console/tutorials/authentication.md:123]: This should be in the `ash-cli` tab

File: docs/console/tutorials/authentication.md [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+
+```bash title="Command"
+ash console auth refresh-token
+```
+
+```bash title="Output"
+Refreshing access token for the Ash Console at http://localhost:8080
+
+Access token refreshed successfully!
+```
```
Comment (leopaul36) [inline docs/console/tutorials/authentication.md:121]: This should be outside the `tip` (like in the `ash-api` tab)

**Review by leopaul36 — APPROVED** [review]

File: docs/console/tutorials/authentication.md (hunk 1 above)
Comment (Nuttymoon) [reply docs/console/tutorials/authentication.md:123]: I don't think so because direct usage of access tokens is not part of the Ash CLI workflow

Comment (leopaul36) [reply docs/console/tutorials/authentication.md:123]: It just feels weird to have `ash` commands in a "Using the API" tab when there is a "Using the CLI" tab IMO

**Pull request merged**